app/Http/Controllers/UserController.php

```diff
@@ -265,14 +265,11 @@ public function loginUser(Request $request, Validate $validate)
             $password = $request->password;
             $user = User::where('email', '=', $username)->first();
 
-            if(empty($user) && !Hash::check($password, $user->password)){
-                $res = (object)[
-                    "status_code" => 401,
-                    "message"     => "Email or password does not match",
-                    "error"       => null,
-                    "data"        => null
-                ];
-                return (new ErrorResource($res))->response()->setStatusCode(401);
+            if(empty($user)){
+                return $this->resProvider->apiJsonResponse(401, "Emails is not registered with us!", null, null);
```

Comment:
@ashish76 The message `Emails is not registered with us` must come from the messages under the Config.

Reply:
And also it should be `Email` instead of Emails